src/components/Button.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import styled from 'styled-components'
+import buttonBG from '../assets/hc_button.svg'
+import { hexToRgba } from '../utility/utilities'
+
+const buttonSize = {
+  small: '60px',
+  default: '80px',
+  large: '300px',
+}
+
+const buttonColors = ['primary', 'secondary', 'tertiary']
+
+const StyledButton = styled.button`
+  display: block;
+  text-decoration: none;
+  font-family: 'HK Grotesk';
+  font-weight: ${p => p.theme.typography.h2.weight};
+  border: transparent;
+  transition: all 250ms;
+  min-width: ${p => (p.size ? buttonSize[p.size] : buttonSize['default'])};
+  text-align: center;
+  padding: 0.75em;
+  border-radius: 3px;
+  margin: 1em;
+  :hover {
+    cursor: pointer;
+  }
+  :focus {
+    outline: 0;
+  }
+  :disabled {
+    cursor: not-allowed;
+    opacity: ${p => p.theme.opacity.disabled};
+  }
+  ${p =>
+    (p.color === buttonColors[0] || !buttonColors.includes(p.color)) && // primary color (gradient button) or not specified color
```

Comment:
maybe? (see constants comment below)
```suggestion
    (p.color === "primary" || !p.color) && // primary color (gradient button) or not specified color
```

Reply:
oop yes update: got rid of constants cuz primary secondary are clear enough